src/formulate.meta/Constants.cs

```diff
@@ -17,7 +17,7 @@ public class Constants
         /// Do not reformat this code. A grunt task reads this
         /// version number with a regular expression.
         /// </remarks>
-        public const string Version = "3.5.2";
+        public const string Version = "3.5.3";
```

Comment:
A version increment should not be in this commit. Those should be in atomic commits specifically for version increments. I will handle this once I've merged in your PR, so feel free to remove this part of your change.